tsl/src/continuous_aggs/create.c

```diff
@@ -186,9 +186,15 @@ typedef struct CAggTimebucketInfo
 typedef struct AggPartCxt
 {
 	struct MatTableColumnInfo *mattblinfo;
-	bool addcol;
+	bool added_aggref_col;
+	bool inside_aggref;			/* Set to true when traversing within an Aggref node */
+	bool var_outside_of_aggref; /* Set to true when you come across a Var that is not inside an
+								   Aggref node */
 	Oid ignore_aggoid;
 	int original_query_resno;
+	List *orig_vars; /* List of Var nodes that have been mapped to materialization table columns */
```

Comment:
These comments don't seem to make sense to me. Can we clarify them further?

Reply:
Added clarifications.